interface/src/app/gene/searchMany/gene-search-panel.tpl.html

```diff
@@ -0,0 +1,17 @@
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <h3>
+      Search for Genes
+    </h3>
+  </div>
+  <div class="panel-body">
+    <div class="col-xs-3">
+      <gene-search-form search-results="searchResults" queries="queries">
+      </gene-search-form>
+    </div>
+    <div class="col-xs-9">  
+      <search-result-table search-results="searchResults" queries="queries">
+      </search-result-table>
+    </div>
+  </div>
+</div>
```

Comment:
I really like how this is now tightly focused solely on layout of these two widgets. We should strive for this clean separation of responsibilities wherever we can. Even the naming (`panel` here vs. `form`, above) is helpfully clear.